### PR TITLE
adding promise polyfill for node 0.10

### PR DESCRIPTION
**Pull request by kellyselden**
fixes ember-cli/ember-cli/pull/4792.

also, kinda depends on #3 to see it working on CI

cc @stefanpenner

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ var styleCheck = /\srel=["\'][^"]*stylesheet[^"]*["\']/;
 var srcCheck = /\ssrc=["\']([^"\']+)["\']/;
 var hrefCheck = /\shref=["\']([^"\']+)["\']/;
 
+// node 0.10
+require('es6-promise').polyfill();
```
Comment (stefanpenner) [inline index.js]: we need to be careful not to force `Promise` to be present, that isn't going to make users happy.

Any lib that uses Promise, should just include some lib (like RSVP and use it directly until we drop drop 10

Comment (kellyselden) [inline index.js]: Yeah you're right, I shouldn't have modified global scope. Oversighted. I will change.

**Comment by kellyselden** [comment]
updated

**Comment by stefanpenner** [comment]
:+1:

**Comment by jonathanKingston** [comment]
:+1: LGTM thank you

**Pull request merged**